roles/ansible-keylime-tpm20/tasks/swtpm.yml

```diff
@@ -0,0 +1,19 @@
+- name: Create swtpm directory
```

Comment:
Does swtpm come with a systemd unit file that performs many of these things at startup? Otherwise a reboot will wipe away the `/tmp/swtpm` directory and the setup will need to be redone again.